### PR TITLE
Lesson 10

### PR DESCRIPTION
**Pull request by Anastasia-Tumilovich**
added a new section rewrote styles to Sass

### CODE REVIEW

File: Triphouse-project/index.html [hunk 1]
```diff
@@ -115,38 +114,208 @@ <h2 class="section__title offer__title"> What do we offer</h2>
...
+                    <button class="active__tab">Regions</button>
+                    <button>Cities</button>
+                    <button class="destinatiom__desktop_button">Places of interest</button>
+                    <button class="destination__mobile_button">Places</button>
+                </div>
+                <ul class="places">
+                    <li class="places__item">
+                        <div>
+                            <button class="book">Book now</button>
+                            <img src="./images/maldives.png" alt="Maldives"/>
```
Comment (ghost) [inline Triphouse-project/index.html:160]: 2021-05-02_21h05_25
2021-05-02_21h05_55

File: Triphouse-project/index.html [hunk 2]
```diff
@@ -5,11 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
```
Comment (ghost) [inline Triphouse-project/index.html:5]: 2021-05-02_21h07_52
Good solution, keep up the good work.

File: Triphouse-project/index.html (hunk 1 above)
Comment (Anastasia-Tumilovich) [reply Triphouse-project/index.html:160]: done

File: Triphouse-project/index.html (hunk 2 above)
Comment (Anastasia-Tumilovich) [reply Triphouse-project/index.html:5]: thank you

**Pull request merged**